actix-http/src/body/boxed.rs

```diff
@@ -14,8 +14,16 @@ use crate::Error;
 pub struct BoxBody(Pin<Box<dyn MessageBody<Error = Box<dyn StdError>>>>);
 
 impl BoxBody {
-    /// Boxes a `MessageBody` and any errors it generates.
+    /// Same as `MessageBody::boxed`.
     pub fn new<B>(body: B) -> Self
+    where
+        B: MessageBody + 'static,
+    {
+        body.boxed()
+    }
+
+    /// Boxes a `MessageBody` and any errors it generates.
+    pub fn new_raw<B>(body: B) -> Self
```

Comment:
I don't get the new method, wouldn't it be okay to just use `BoxBody::new` in the default `.boxed` impl ?